src/osd/PGLog.h

```diff
@@ -588,7 +597,7 @@ struct PGLog : DoutPrefixProvider {
   }
 public:
   bool is_dirty() const {
-    return !touched_log ||
+    return dirty_log ||
```

Comment:
we should not be removing `!touched_log`, rather use `!touched_log` || `dirty_log` ||

Reply:
ack - thanks @neha-ojha